installers/olm/openshift.description.md

```diff
@@ -34,166 +34,112 @@ and much more!
 [pgBackRest]: https://www.pgbackrest.org
 [pgMonitor]: https://github.com/CrunchyData/pgmonitor
 
+
 ## Before You Begin
 
-There are several manual steps that the cluster administrator must perform prior to installing the operator. The
-operator must be provided with an initial configuration to run in the cluster, as well as certificates and
-credentials that need to be generated.
+There are a few manual steps that the cluster administrator must perform prior to installing the PostgreSQL Operator.
+At the very least, it must be provided with an initial configuration.
 
-Start by cloning the operator repository locally.
+First, select a namespace in which to install the PostgreSQL Operator. PostgreSQL clusters will also be deployed here.
+If it does not exist, create it now.
 
 ```
-git clone -b v${PGO_VERSION} https://github.com/CrunchyData/postgres-operator.git
-cd postgres-operator
+export PGO_OPERATOR_NAMESPACE=pgo
+oc create namespace "$PGO_OPERATOR_NAMESPACE"
 ```
 
-### PostgreSQL Operator Configuration
-
-Edit `conf/postgres-operator/pgo.yaml` to configure the operator deployment. Look over all of the options and make any
-changes necessary for your environment.
-
-#### Image
-
-Update the `CCPImageTag` tag to configure the PostgreSQL image being used, updating for the version of PostgreSQL as needed.
+Next, clone the PostgreSQL Operator repository locally.
 
 ```
-CCPImageTag:  ${CCP_IMAGE_TAG}
+git clone -b v${PGO_VERSION} https://github.com/CrunchyData/postgres-operator.git
+cd postgres-operator
 ```
 
-#### Storage
+### Security
 
-Configure the backend storage for the Persistent Volumes used by each PostgreSQL cluster. Depending on the type of persistent
-storage you wish to make available, adjust the `StorageClass` as necessary. For example, to deploy on AWS using `gp2`, you
-would set the following:
+Install the Security Context Constraint for OpenShift:
 
 ```
-storageos:
-  AccessMode:  ReadWriteOnce
-  Size:  1G
-  StorageType:  dynamic
-  StorageClass:  gp2
-  Fsgroup:  26
+oc create -f ./deploy/pgo-scc.yaml
```

Comment:
Perhaps due to the hubbub with SCCs lately, we should include a note around using this vs. running in restricted mode?

Reply:
Replaced with `DisableFSGroup: true`.